### ISSUE TITLE
Unexpected token :

### ISSUE THREAD

**Issue by ipselon** [royriojas/esformatter-jsx#16]
esformatter verision: 0.7.3, esformatter-jsx version: 2.0.10

```
var React = require('react');
var Button = require('react-bootstrap');
var Testing = React.createClass({

    getDefaultProps: function () {
        return {};
    },

    render: function(){
        return (
            <div {...this.props} >

                    <h3 style={{  padding: '1em', textAlign: 'center' }} ><span >This is an empty page. To add new component select needed element on left-side panel and click on an element on the page where you want to put new component.</span></h3>

                    <Button bsStyle={'primary'} ><span >Text</span></Button>


            </div>
        );
    }
});

module.exports = Testing;
```

**Comment by royriojas** [comment]
Interesting...

In the meantime, here is a workaround, just in case

```javascript
var React = require('react');
var Button = require('react-bootstrap');
var Testing = React.createClass({

    getDefaultProps: function () {
        return {};
    },

    render: function(){
        var style = {  padding: '1em', textAlign: 'center' };
        return (
            <div {...this.props} >

                    <h3 style={style} ><span >This is an empty page. To add new component select needed element on left-side panel and click on an element on the page where you want to put new component.</span></h3>

                    <Button bsStyle={'primary'} ><span >Text</span></Button>


            </div>
        );
    }
});

module.exports = Testing;
```

**Comment by royriojas** [comment]
Fixed in v2.0.11.

ObjectExpression and ArrayExpression are forced to be in a single line inside the JSXExpressionContainer.

While this works well for small objects, it might not be practical for big ones.

Regards
